documentation/contributing/code/index.markdown

```diff
@@ -54,11 +54,13 @@ Please note that we rely on clang-format version **3.8**. Sadly, newer versions
 
 ### Command Line
 
-Format single file:
+Add [this](https://raw.githubusercontent.com/davetcoleman/roscpp_code_format/master/.clang-format) format file at ~/catkin_ws/.clang-format. Now any file inside your catkin workspace will be formatted with the ROS style guidelines described in this config file.
```

Comment:
(1) for moveit, we have the file as part of the git repository. I don't want to point users to Dave's private repository here.
(2) `~/catkin_ws` is not always the correct workspace. Why do you want to add it to the workspace anyway? This documentation is about MoveIt. I do not want to use this style for my private code.

Reply:
I see. How about a short reference to the ./clang-format file so new contributors at least know it exists, then.

Reply:
sounds good to me.